cl/search/tests/tests_es_recap.py

```diff
@@ -864,6 +864,42 @@ def test_filter_docket_with_no_documents(self) -> None:
             docket.delete()
             docket_2.delete()
 
+    def test_cause_filter(self) -> None:
+        """Confirm cause filter works properly"""
+        # Confirm parties extracted from case_name are available in filters.
+        with self.captureOnCommitCallbacks(execute=True):
+            d = DocketFactory(
+                court=self.court,
+                docket_number="23-cv-12335",
+                case_name="Lockhart v. Gainwell Technologies LLC",
+                cause="31:3730 Qui Tam False Claims Act",
+                source=Docket.RECAP,
+            )
+            d_2 = DocketFactory(
+                court=self.court,
+                docket_number="22-cv-00526",
+                case_name="Schermerhorn v. Quality Enterprises USA, Inc.",
+                cause="31:3730 Qui Tam False Claims Act",
+                source=Docket.RECAP,
+            )
+
+        params = {
+            "type": SEARCH_TYPES.RECAP,
+            "q": 'cause:"31:3730 Qui Tam False Claims Act"',
+        }
+        async_to_sync(self._test_article_count)(
+            params, 2, "faceted_cause_query_string"
+        )
+        params = {
+            "type": SEARCH_TYPES.RECAP,
+            "cause": '"31:3730 Qui Tam False Claims Act"',
+        }
+        async_to_sync(self._test_article_count)(params, 2, "cause_filter")
```

Comment:
I think it'd be helpful to pull the cause string out to show that it's the same in every instance:

```suggestion
        cause_str = "31:3730 Qui Tam False Claims Act"
        params = {
            "type": SEARCH_TYPES.RECAP,
            # Do it in main query box
            "q": f'cause:"{cause_str}"',
        }
        async_to_sync(self._test_article_count)(
            params, 2, "faceted_cause_query_string"
        )
        params = {
            "type": SEARCH_TYPES.RECAP,
            # Do it in the cause field as a phrase
            "cause": f'"{cause_str}"',
        }
        async_to_sync(self._test_article_count)(params, 2, "cause_filter")
        params = {
            "type": SEARCH_TYPES.RECAP,
            # Do it in the cause field without quotes
            "cause": f'{cause_str}',
        }
        async_to_sync(self._test_article_count)(
            params, 2, "cause_filter_no_phrase"
        )
```

I also added a test without quotes, which I hope will work?

Reply:
> I think it'd be helpful to pull the cause string out to show that it's the same in every instance:

Great idea. I'll update the code to reflect this change.